.github/workflows/ci-build.yml

```diff
@@ -11,34 +11,49 @@ on:
 
 jobs:
   test:
-    name: Tests
+    name: "Tests"
```

Comment:
Does this need to be quoted?

Reply:
Not sure tbh, I think works either way, but better revert to align with addon blueprint